kubeflow/common/knative/kustomization.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
```

Comment:
Question: is `kubeflow/common/knative/knative-1-2-5/serving-crds.yaml` being used anywhere? Should we remove it?

Reply:
`serving-core.yaml` includes `serving-crds.yaml`. To install knative, however, it is recommended to apply `serving-crds.yaml` before applying `serving-core.yaml`. It doesn't apply to us as we handle it in the Makefile, but it worth keeping `serving-crds.yaml` for reference and to support users who have follow knative installation instructions and depend on `serving-crds.yaml`. I added a comment inside kustomize.yaml to clarify this.